kiwi/markup/base.py

```diff
@@ -88,3 +103,16 @@ def get_yaml_description(self) -> str:
         Implementation in specialized Markup class
         """
         raise NotImplementedError
+
+
+class FileResolver(etree.Resolver):
+    def resolve(self, url, pubid, context):
+        if os.path.exists(url):
+            return self.resolve_filename(url, context)
+        else:
+            # In case a document() reference file cannot be found
+            # an <include> statement is created to be consumed by
+            # the check_include_references_unresolvable runtime check
+            return self.resolve_string(
```

Comment:
I am fine with this code. I am trying to understand its motivation though. Correct me if wrong, we are not throwing an exception here for two reasons:

* Provide a got XML schema error if any (unrelated with the current include)
* Provide a proper list of missing includes on runtime check once the schema (without the missing include) is already validated.

Just asking because to my eyes throwing an exception here could save us some lines of code at the expense of not having a full schema validation.

Reply:
I think this will be addressed from the latest commit I did